patroni/config.py

```diff
@@ -601,3 +602,20 @@ def get_global_config(self, cluster: Union[Cluster, None]) -> GlobalConfig:
         :returns: :class:`GlobalConfig` object
         """
         return get_global_config(cluster, self._dynamic_configuration)
+
+    def _validate_failover_tags(self):
+        """Checks nofailover/failover_priority config and warns user if it's contradictory"""
+        nofailover_tag = self.get('tags', {}).get('nofailover', None)
+        failover_priority_tag = self.get('tags', {}).get('failover_priority', None)
+        if nofailover_tag is not None and failover_priority_tag is not None:
+            if (
+                nofailover_tag is True and failover_priority_tag > 0
+                or nofailover_tag is False and failover_priority_tag <= 0
+            ):
+                logger.warning(
+                    'Conflicting configuration between nofailover: %s and failover_priority: %s.'
+                    + ' Defaulting to nofailover: %s',
+                    nofailover_tag,
+                    failover_priority_tag,
+                    nofailover_tag
+                )
```

Comment:
yes exactly. I updated the `Defaulting to nofailover: %s` to accurately reflect case where `nofailover: False` and `failover_priority: 0`, but I think we should still log in this case since it's an undesirable configuration to begin with

Reply:
just to clarify, what do we want to happen when provided:
`
nofailover: False,
failover_priority: 0
`?

Should this be the same as `nofailover: False, failover_priority: 1` or `nofailover: False, failover_priority: 0`?

Reply:
> Should this be the same as nofailover: False, failover_priority: 1 or nofailover: False, failover_priority: 0?

IMO, the second seems to be more logical. Why should failover_priority suddenly change its value?